audio/beets/Portfile

```diff
@@ -0,0 +1,517 @@
+}
+
+subport beets-copyartifacts {
+    name            beets-copyartifacts
+    python.rootname beets-copyartifacts3
+    version         0.1.3
+    revision        0
+
+    license         MIT
+
+    description     beets plugin to copy non-music files to import path
+    long_description ${description}
+
+    homepage        https://github.com/adammillerio/beets-copyartifacts
+
+    checksums       rmd160  0a3295d358bf353247fdb1911b3a7656a7beb9e5 \
+                    sha256  e9fb32f3ec5dc279c96bd010322e0792e55e84eecf0c3383f74ef2b624539969 \
+                    size    5452
+}
+
+subport beets-describe {
+    name            beets-describe
+    version         0.0.3
+    revision        0
+
+    license         MIT
+
+    description     A beets plugin that describes attributes in depth
+    long_description ${description}
+
+    homepage        https://github.com/adamjakab/BeetsPluginDescribe
+
+    checksums       rmd160  fa35987e55e63e1090318dc79bd95b4cec5459ed \
+                    sha256  4f47bd8b4b3852b4d9f4b698f0727ddd75a813e9c528100ee20876df6d6227f0 \
+                    size    8398
+
+    depends_lib-append \
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-pandas \
+                    port:py${python.version}-termplotlib \
+                    port:py${python.version}-termtables
+}
+
+subport beets-follow {
+    PortGroup       github 1.0
+
+    github.setup    nolsto beets-follow eb504a0b3b457993d599530a66a54b11b740ecb5
+    version         20210908
+    revision        0
+
+    license         MIT
+
+    description     Plugin for the music library manager Beets. Follow artists from your library using muspy.com
+    long_description ${description}
+
+    homepage        https://github.com/nolsto/beets-follow
+
+    checksums       rmd160  ebe6866b1055ea5459fa33c77b08f904dece5be8 \
+                    sha256  e1dd35c3ea1cf8c3f5893fa3b49f3fd2f1c1c230e05f5fa0bf03b805a9eea39a \
+                    size    4145
+
+    patch.pre_args  -p1
+    patchfiles      follow-confuse.diff
+
+    depends_lib-append \
+                    port:py${python.version}-confuse
+}
+
+subport beets-goingrunning {
+    name            beets-goingrunning
+    version         1.2.2
+    revision        0
+
+    license         MIT
+
+    description     A beets plugin for creating and exporting songs that match your running session.
+    long_description ${description}
+
+    homepage        https://github.com/adamjakab/BeetsPluginGoingRunning
+
+    checksums       rmd160  e761fda8ec710d6df6b930e502432c1ce9751646 \
+                    sha256  1facf57accbc77329831ceff15ffcbf024de40fa4826f63ae7987607ebf0c1d5 \
+                    size    22371
+
+    depends_lib-append \
+                    port:py${python.version}-alive-progress
+}
+
+subport beets-ibroadcast {
+    name            beets-ibroadcast
+    version         0.5.0
+    revision        0
+
+    license         Unlicense
```

Comment:
ideally all licenses should be recognized by MacPorts (see https://github.com/macports/macports-infrastructure/blob/master/jobs/distributable_lib.tcl); I don't think this one is... But I *think* that `public-domain` would be an appropriate choice here

Reply:
you didn't agree with the suggestion to use `public-domain`?